client/components/product-details.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+export default class ProductDetails extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { product: null };
+    this.setViewCat = this.setViewCat.bind(this);
+  }
+
+  componentDidMount() {
+    fetch(`/api/products/${this.props.viewParams.productId}`)
+      .then(reponse => reponse.json())
+      .then(data => {
+        return this.setState({ product: data });
+      });
+  }
+
+  setViewCat(event) {
+    this.props.setView('catalog', {});
+  }
+
+  render() {
+    if (!this.state.product) return null;
+    return (
+      <div className="row mt-5 mx-0">
+        <div className="details card col-11 mx-auto" id={this.props.productId}>
+          <div className="hover text-muted my-3 px-0 btn d-flex justify-content-start" onClick={this.setViewCat} style={{ cursor: 'pointer' }}>&lt; Back to catalog</div>
+          <div className="row">
+            <img
+              src={this.state.product.image}
+              className="card-img-top col-4"
+              alt={this.state.product.name}
+              style={{ objectFit: 'contain', maxHeight: '300px' }} />
```

Comment:
It's best to keep your styling in a class in a css file, in case you ever need to adjust it.